### PR TITLE
GroupBy option for line charts

### PR DESCRIPTION
**Pull request by KhadijaMahanga**
## Description

- Activate `groupBy` option for line charts
- Upgrade hurumap-ui version

Fixes #[171123754](https://www.pivotaltracker.com/story/show/171123754)

## Type of change
- [x] New feature (non-breaking change which adds functionality)

## Desktop Screenshots

Peek 2020-02-07 15-26

Screenshot from 2020-02-07 10-14-45

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by KhadijaMahanga** [comment]
@kilemensi can we get advised on line chart color scale from design team? The current scale seems too close to each other and does not differentiate well grouped line

**Review by karimkawambwa — APPROVED** [review]
LGTM 💯 

Maybe design will give us better color ranges.

File: wp-content/plugins/hurumap/src/Theme.js [hunk 1]
```diff
@@ -39,6 +39,14 @@ export default createTheme({
         }
       }
     },
+    line: {
+      colorScale: ['#29a87c', '#a0d9b3', '#004494', '#4abc70'],
```
Comment (kilemensi) [inline wp-content/plugins/hurumap/src/Theme.js:43]: Will these colours also be available in [Takwimu.AFRICA](/TakwimuAfrica/Takwimu.AFRICA)

File: wp-content/plugins/hurumap/src/ChartDefinition/HurumapChart.js [hunk 2]
```diff
@@ -229,6 +229,32 @@ function HurumapChartDefinition({ chart, data, sectionOptions, onChange }) {
...
+                <Grid
+                  item
+                  container
+                  component="label"
+                  alignItems="center"
+                  spacing={1}
+                >
+                  <Grid item>Interpolation: Linear</Grid>
+                  <Grid item>
+                    <Switch
```
Comment (kilemensi) [inline wp-content/plugins/hurumap/src/ChartDefinition/HurumapChart.js]: Lets switch to a dropdown with all the options: https://formidable.com/open-source/victory/docs/victory-line/#interpolation

File: wp-content/plugins/hurumap/src/Theme.js [hunk 3]
```diff
@@ -50,7 +58,9 @@ export default createTheme({
           fontFamily: FONT_FAMILY_TEXT,
           fill: 'rgb(0,0,0)'
         }
-      }
+      },
+      fixLabelOverlap: true,
+      tickCount: 6
```
Comment (kilemensi) [inline wp-content/plugins/hurumap/src/Theme.js]: 6 would still be a bit high for HURUmap data e.g. when the values are in millions. We're using 3 in UI: https://github.com/TakwimuAfrica/Takwimu.AFRICA/blob/db745881ca9b32d387bce79d06e4a96e721589ea/src/theme.js#L61

**Review by kilemensi — APPROVED** [review]
:rocket: @KhadijaMahanga

File: wp-content/plugins/hurumap/src/Theme.js (hunk 1 above)
Comment (KhadijaMahanga) [reply wp-content/plugins/hurumap/src/Theme.js:43]: @kilemensi , yes I will add them there. These are from takwimu style guide

File: wp-content/plugins/hurumap/src/Theme.js (hunk 3 above)
Comment (KhadijaMahanga) [reply wp-content/plugins/hurumap/src/Theme.js]: I have reduce this to 4

File: wp-content/plugins/hurumap/src/ChartDefinition/HurumapChart.js (hunk 2 above)
Comment (KhadijaMahanga) [reply wp-content/plugins/hurumap/src/ChartDefinition/HurumapChart.js]: Cool

**Pull request merged**